### PR TITLE
Add ConsenSys to Adopters

### PR DESCRIPTION
**Pull request by clementblaise**
### Description of your changes

Fixes #

I have:

- [ ] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

PS: left editable in case of conflict

### CODE REVIEW

**Comment by jbw976** [comment]
Great to see your name making this contribution @clementblaise! 👋 😎 

We're merging these PRs in the order that they were opened, so once #3739 has been merged then I'll ping you back on this PR here to coordinate next steps and get yours merged too :)

**Comment by jbw976** [comment]
Oh nice, I see you have "Maintainers are allowed to edit this pull request"  checked and explicitly mentioned in the PR body, so I'll probably just take care of the merge conflict on your behalf (you still get the author/contributor credit 😉) to relieve you of that burden 😅

File: ADOPTERS.md
Comment (jbw976) [inline ADOPTERS.md:38]: Is this IDP seeing production usage too? If so, it may be nice to explicitly mention that, e.g. "Main control plane for an internal developer platform being used in production" 🙏

Comment (clementblaise) [reply ADOPTERS.md:38]: Thanks @jbw976 ! Our production only hosts development workloads for now, some should hit production in a few weeks. I will let you be the judge 😉

Comment (jbw976) [reply ADOPTERS.md:38]: i'll keep it honest for now hehe 😅 - feel free to keep us updated in the future when you progress your deployments!! 🙏

**Review by jbw976 — APPROVED** [review]

**Comment by jbw976** [comment]
This is all merged in now @clementblaise, thanks again for sharing your Crossplane adoption with the community!! 🙇 👏

**Pull request merged**
